### DISCUSSION TITLE
Add Code Tag

### DISCUSSION THREAD

**Discussion by ThePowerTool** [community/community#51890] [New to GitHub] [answered]
Hi!  This is my first forum post, here.  I've been aroung some time but am mainly a rote-cloner (I know how to clone and compile, to some extent).

I have always made an effort to contribute and have run into a question regarding posts.

When using the "Add Code" tag I'm used to seeing the added code in a box with a slightly different background and each character unchanged from the paste.  I'm also used to either not seeing the actual tag (meaning just the GUI indicating code during input/creation) or the tag being sophisticated-enough to minimize issues with content.

I have noticed when posting code I frequently run into issues with the characters used for the add-code designators being in the code.

Is there a page with instructions on "creating posts" and the editing options that details how the "add code" button is used?  I've tried searching but no luck.

Any pointers would be greatly appreciated.

Thanks!

**Comment by airtower-luna** [comment, marked as answer]
If you mean for writing here and in issues: Use Markdown. Markdown implementations vary a bit in advanced features, but code blocks and other basic formatting are the same.

The GitHub documentation describes their flavor of Markdown, start here: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

**Comment by ThePowerTool** [comment]
This is a test:
```
    psql_major_version=`$PG_CONFIG $psql_args --version | \ 
           sed 's/PostgreSQL \([0-9]*\).\([0-9]*\)\(.\([0-9]*\)\).*$/\1/'`
    psql_minor_version=`$PG_CONFIG $psql_args --version | \
           sed 's/PostgreSQL \([0-9]*\).\([0-9]*\)\(.\([0-9]*\)\).*$/\2/'`
    psql_micro_version=`$PG_CONFIG $psql_config_args --version | \
           sed 's/PostgreSQL \([0-9]*\).\([0-9]*\)\(.\([0-9]*\)\).*$/\4/'`
```

**Comment by ThePowerTool** [comment]
That is exactly what I was looking for.  Thank you very much for the pointer!!!!
